source/lmp/tests/test_lammps_dpa3_pt2.py

```
@@ -315,3 +415,489 @@ def test_pair_deepmd_si(lammps_si) -> None:
+            str(pb_file_mpi.resolve()),
+            out_path,
+        ]
+        if processors is not None:
+            argv.extend(["--processors", processors])
+        elif nprocs == 1:
+            argv.extend(["--processors", "1 1 1"])
+        if extra_args:
+            argv.extend(extra_args)
+        if runner_args:
+            argv.extend(runner_args)
+        sp.check_call(argv)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)